src/components/ApyCalculatorModal/index.tsx

```diff
@@ -8,6 +8,7 @@ interface ApyCalculatorModalProps {
   onDismiss?: () => void
   tokenPrice: number
   apr: number
+  combinedApr?: string
```

Comment:
A small thing but maybe we should change the name of this. In other places your variable names ending `Apr` are numbers.

Alternatively, make this a number.

Reply:
Maybe add the word `display` like we do in other places.

Reply:
Changed to `displayApr`, what do you think?